### PR TITLE
Feature/1

### PR DESCRIPTION
**Pull request by icerrate**

### CODE REVIEW

File: app/src/main/kotlin/com.mutenlab.sudoit/SplashActivity.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    override fun onResume() {
+        super.onResume()
+        mDelayHandler!!.postDelayed(mRunnable, SPLASH_DELAY)
+    }
+
+    override fun onPause() {
+        if (mDelayHandler != null) {
+            mDelayHandler!!.removeCallbacks(mRunnable)
+        }
+        super.onPause()
```
Comment (EduardoChuks) [inline app/src/main/kotlin/com.mutenlab.sudoit/SplashActivity.kt]: the super method should be called at beginning.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -3,19 +3,20 @@
...
     <application
-        android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity
+            android:name=".SplashActivity">
```
Comment (EduardoChuks) [inline app/src/main/AndroidManifest.xml]: Remember that if the splash screen uses an especific background color or an immutable image, it should be set as theme and not in the activity layout.

Comment (icerrate) [reply app/src/main/AndroidManifest.xml]: Sure thing. Applying changes

File: app/src/main/kotlin/com.mutenlab.sudoit/SplashActivity.kt [hunk 3]
```diff
@@ -22,8 +22,8 @@ class SplashActivity : AppCompatActivity() {
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        setTheme(R.style.SplashTheme)
```
Comment (EduardoChuks) [inline app/src/main/kotlin/com.mutenlab.sudoit/SplashActivity.kt]: It is not necessary to set theme in activity if you already did on Manifest.

**Review by EduardoChuks — APPROVED** [review]
Code looks good 👍

**Pull request merged**